.github/dependabot.yml

```diff
@@ -13,6 +13,10 @@ updates:
       interval: "daily"
     pull-request-branch-name:
       separator: "-"
+    ignore:
+      # We currently need to ignore eslint@v9
```

Comment:
@nmerget could you please even also mention the reason and/or provide a GH issue so that we could find & refactor this one later on ?